docs/source/flow_setup.rst

```diff
@@ -1,4 +1,22 @@
-Setup Instructions
+.. contents:: Table of contents
+
+1 - Local Installation
```

Comment:
The numbering scheme here is different than the numbering scheme below. Is this correct @lucfisc